uber/utils.py

```diff
@@ -477,6 +477,31 @@ def get_real_badge_type(badge_type):
     return c.ATTENDEE_BADGE if badge_type in [c.PSEUDO_DEALER_BADGE, c.PSEUDO_GROUP_BADGE] else badge_type
 
 
+def add_opt(opts, other):
+    """
+    Add an option to a comma-separated string.
+    """
+    other = [other] if isinstance(other, str) else other
+    list = opts.split(',') if opts else []
+    list.extend(other)
+    return ','.join(list)
+
+
+def remove_opt(opts, other):
+    """
+    Remove an option from a comma-separated string.
+    """
+    other = [other] if isinstance(other, str) else other
+    list = opts.split(',')
+    if other == list:
+        return ''
+    elif other not in list:
```

Comment:
The easiest way to remove the items in one list from another is to use the `set().difference()` method!

For `remove_opts()` we can make similar changes to what we made in `add_opts()`:

```python
def remove_opt(opts, other):
    other = listify(other) if other else []
    other = [str(s) for s in other]  # support both int and str for other
    list = set(opts.split(','))
    return ','.join(list.difference(other))
```